mobile-app/cypress/integration/functional/wallet/balances/balances.spec.ts

```diff
@@ -259,3 +259,146 @@ context('Wallet - Balances - No balance', () => {
+      name: 'Playground BTC-Default Defi token',
+      status: true,
+      tokenA: {
+        symbol: 'BTC',
+        displaySymbol: 'dBTC',
+        id: '1',
+        reserve: pair1ReserveA,
+        blockCommission: '0'
+      },
+      tokenB: {
+        symbol: 'DFI',
+        displaySymbol: 'DFI',
+        id: '0',
+        reserve: pair1ReserveB,
+        blockCommission: '0'
+      },
+      priceRatio: {
+        ab: '1',
+        ba: '1'
+      },
+      commission: '0',
+      totalLiquidity: {
+        token: '2500',
+        usd: '20000000'
+      },
+      tradeEnabled: true,
+      ownerAddress: 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy',
+      rewardPct: '0.1',
+      creation: {
+        tx: '79b5f7853f55f762c7550dd7c734dff0a473898bfb5639658875833accc6d461',
+        height: 132
+      },
+      apr: {
+        reward: 66.8826,
+        total: 66.8826
+      }
+    },
+
+    {
+      id: '20',
+      symbol: 'DUSD-DFI',
+      displaySymbol: 'DUSD-DFI',
+      name: 'Decentralized USD-Default Defi token',
+      status: true,
+      tokenA: {
+        symbol: 'DUSD',
+        displaySymbol: 'DUSD',
+        id: '14',
+        reserve: pair2ReserveA,
+        blockCommission: '0'
+      },
+      tokenB: {
+        symbol: 'DFI',
+        displaySymbol: 'DFI',
+        id: '0',
+        reserve: pair2ReserveB,
+        blockCommission: '0'
+      },
+      priceRatio: {
+        ab: '10',
+        ba: '0.1'
+      },
+      commission: '0.02',
+      totalLiquidity: {
+        token: '2500',
+        usd: '16660'
+      },
+      tradeEnabled: true,
+      ownerAddress: 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy',
+      rewardPct: '0.1',
+      creation: {
+        tx: '4b8d5ec122052cdb8e8ffad63865444a10edc396d44e52957758ef7a39b228fa',
+        height: 147
+      },
+      apr: {
+        reward: 80291.23649459783,
+        total: 80291.23649459783
+      }
+    }
+  ]
+
+  before(function () {
+    cy.intercept('**/poolpairs?size=*', {
+      body: {
+        data: getChangingPoolPairReserve({
+          pair1ReserveA: '1001',
+          pair1ReserveB: '1001',
+          pair2ReserveA: '8330',
+          pair2ReserveB: '830'
+        })
+      }
+    })
+    cy.createEmptyWallet(true).sendDFItoWallet().sendTokenToWallet(['BTC', 'DUSD-DFI']).wait(3000)
+  })
+
+  it('should be able to get DEX Price USD Value', function () {
+    cy.getByTestID('balances_row_1_amount').should('have.text', '10.00000000')
+    // (1001 / 1001) * (8330 / 830) * 10.00
+    cy.checkBalanceRow('1', { name: 'Playground BTC', amount: '10.00000000', displaySymbol: 'dBTC', symbol: 'BTC', usdAmount: '≈ $100.36' })
+    // DUSD  = ((10 / 2500) * 8330) * 1
+    // DFI =  ((10 / 2500) * 830) * (8330 / 830)
+    // DFI + DUSD
+    cy.checkBalanceRow('20', { name: 'Decentralized USD-DeFiChain', amount: '10.00000000', displaySymbol: 'DUSD-DFI', symbol: 'DUSD-DFI', usdAmount: '≈ $66.64' })
+  })
+
+  it('should be able display updated Dex Price USD Value', function () {
```

Comment:
Can you add this scenario? When you receive a token (trigger a Send), check if the DEX prices will update

Reply:
Updated!  
![image](https://user-images.githubusercontent.com/13292662/151100716-845dc4c7-d786-4fc2-b060-5af873edfc96.png)